### ISSUE TITLE
Support timing mutation on animations.

### ISSUE THREAD

**Issue by shans** [web-animations/web-animations-next#328]

**Comment by hgl** [comment]
I'm wondering why is this web-animations-next? If I'm not wrong, level 1 spec defines mutable `AnimationEffectTiming`.

**Comment by dstockwell** [comment]
Yes, it's part of level 1 in the spec. But the polyfill tracks the features that are implemented natively, as native implementations appear we'll move features across.

**Comment by hgl** [comment]
Ah, make sense. So web-animations-1 actually means to polyfill what's already implemented, and web-animations-next means to polyfill what's specced (either level one or two) but not yet natively implemented?

**Comment by dstockwell** [comment]
Basically, yes. web-animations-1 corresponds to https://github.com/web-animations/web-animations-next#web-animationsminjs -- eventually it should contain everything in level 1.

**Comment by hgl** [comment]
Got it. Thanks.
